element_array_ephys/ephys.py

```diff
@@ -25,7 +28,8 @@ def activate(ephys_schema_name, probe_schema_name=None, *, create_schema=True,
         :param linking_module: a module name or a module containing the
          required dependencies to activate the `ephys` element:
             Upstream tables:
-                + Session: parent table to ProbeInsertion, typically identifying a recording session
+                + Subject: table referenced by ProbeInsertion, typically identifying the animal undergoing a probe insertion
```

Comment:
`Subject` is not needed in `ephys` and `ephys-no-curation`, only in `ephys-chronic`